addon/serializers/drf.js

```diff
@@ -12,6 +12,40 @@ import Ember from 'ember';
  */
 export default DS.RESTSerializer.extend({
 
+  normalize: function(type, hash, prop) {
+    this.addRelationshipsToLinks(type, hash);
```

Comment:
To support hyperlink related fields, we could search for URLs on relationships using a regex and add them to the 'links' hash.